eslint-config-ofmt/eslint.layout.cjs

```diff
@@ -0,0 +1,24 @@
+/**
```

Comment:
@quesabe I would suggest renaming the file to `eslint.formatting.cjs`, to be extremely explicit about the purpose of the file — it should never include static analysis rules. Also, please add a comment righting the file about the purpose of the file.